src/react-apps/applications/runtime/package.json

```diff
@@ -44,10 +48,7 @@
     "reselect": "3.0.1",
     "uuid": "3.2.1",
     "webpack": "4.12.0",
-    "webpack-dev-server": "3.1.14",
-    "react-content-loader": "4.2.1",
-    "react-dnd": "7.0.2",
-    "react-dnd-html5-backend": "7.0.2"
+    "webpack-dev-server": "3.1.14"
```

Comment:
Found out that some other packages are also placed wrong. Webpack-dev-server is a dev dep. But every package in the react apps is dev deps, because we transpile the app.

Reply:
That does not mean that every dep is a dev dep? When building the app you still rely on some of the packages, while not others. Jest is a dev dep for instance, while moment is used in prod.

Reply:
Every dep is actually a dev dep.

Reply:
No. Why would you want to download jest when you are building for prod? No need at all.